sql/table.go

```diff
@@ -76,9 +78,13 @@ func makeTableDesc(p *parser.CreateTable) (structured.TableDescriptor, error) {
 					ColumnNames: []string{string(d.Name)},
 				}
 				if d.PrimaryKey {
-					index.Name = structured.PrimaryKeyIndexName
+					if index.Name == "" {
```

Comment:
good point. I actually just copied the code below to here. But here it's always going to be true. Will update